docs/merged_objects.md

```diff
@@ -224,8 +224,104 @@ Some merged objects may have some additional sample metadata columns specific to
 Examples of this include treatment or outcome.
 
 
+### SingleCellExperiment dimensionality reduction results
 
+The `reducedDim` slot of the merged object will be contain both principal component analysis (`PCA`) and `UMAP` results.
 
+PCA results were calculated using [`batchelor::multiBatchPCA`](https://rdrr.io/bioc/batchelor/man/multiBatchPCA.html), specifying libraries as batches to ensure that each library in the merged object was equally weighted, and specifying a list of highly variable genes.
+The highly variable genes were selected with `scran::modelGeneVar`, with libraries specified as blocks, and `scran::getTopHVGs`, and they are stored in the `SingleCellExperiment` object in `metadata(merged_sce)$merged_highly_variable_genes`.
```

Comment:
I don't know that we want to use the term `blocks` as that is specific to the argument name for `scran::modelGeneVar`. What about this? 
```suggestion
The highly variable genes were selected in a library-aware manner with `scran::modelGeneVar` and `scran::getTopHVGs`. 
The vector of highly variable genes are stored in the `SingleCellExperiment` object in `metadata(merged_sce)$merged_highly_variable_genes`.
```